src/workerd/io/worker.c++

```diff
@@ -3380,10 +3385,19 @@ void Worker::Actor::setIoContext(kj::Own<IoContext> context) {
   }
   auto& limitEnforcer = context->getLimitEnforcer();
   impl->ioContext = kj::mv(context);
-  impl->metricsFlushLoopTask = impl->metrics->flushLoop(impl->timerChannel, limitEnforcer)
-      .eagerlyEvaluate([](kj::Exception&& e) {
-    LOG_EXCEPTION("actorMetricsFlushLoop", e);
+  impl->onShutdownTask = onShutdown().then(
+      [&context = KJ_ASSERT_NONNULL(this->impl->ioContext)](ShutdownContext shutdown) mutable {
+    if (shutdown.mode == ShutdownMode::HARD) {
+      context->abort(kj::mv(shutdown.error).orDefault(KJ_EXCEPTION(
+          FAILED, "broken.ignored; jsg.internal-error; actor was "
```

Comment:
Dumb question: is this the right error string to be using?  The string `internal-error` only appears here and src/workerd/jsg/exception.h doesn't seem to list internal-error as one of the `JSG_ERROR_` types.